blog-article.html

```diff
@@ -0,0 +1,454 @@
+                <a href="#">
+                    <img src="img/logo-white.png" width="188" height="44" alt="binabox" >
+                </a>
+            </strong>
+            <nav class="navigation">
+                <ul class="navigation__list">
+                    <li class="navigation__item active">
+                        <a href="#" class="navigation__link">
+                            Home
+                        </a>
+                    </li>
+                    <li class="navigation__item">
+                        <a href="/blog-article.html" class="navigation__link">
+                            Blog
+                        </a>
+                    </li>
+                    <li class="navigation__item">
+                        <a href="/catalog.html" class="navigation__link">
+                            Catalog
+                        </a>
+                    </li>
+                    <li class="navigation__item">
+                        <a href="/contact-us.html" class="navigation__link">
+                            Contact us
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="tools">
+                <form class="search">
+                    <button type="submit" class="search__btn">
+                        <img src="img/Icon.svg" width="24" height="24" alt="search"> search
+                        <span class="accessibility">
+                            Search
+                        </span>
+                    </button>
+                    <input type="search" placeholder="Search">
+                </form>
+                <a href="#" class="btn">
+                    <img src="img/discord.svg" width="24" height="24" alt="Discord">
+                    Discord
+                </a>
+                <a href="#" class="btn btn--blue">
+                    <img src="img/wallet.svg" width="24" height="24" alt="Connect">
+                    Connect
+                </a>
+            </div>
+        </div>   
+    </header>
+    <main class="main">
+        <div class="intro">
+            <h1 class="intro__title">
+                Our blog
+            </h1>
+            <ul class="breadcrumbs">
+                <li>
+                    <a href="#">
+                        Home
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        Our blog
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        The Basilisks NFT
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="wrapper">
+            <aside class="sidebar"> perebit classy
+                <div class="sidebar-box">
+                    <strong class="sidebar-title">
+                        Categories
+                    </strong>
+                    <nav class="sidebar__nav">
```

Comment:
Can you please tell me why the nav tag is used here? I don't think it fits here, because we don't have links to internal pages, but there is filtering

And it's very cool that a semantic tag is used, not a div!

Reply:
Because it's not a filter. It's navigation inside different blog articles.
Filter we can see on another page in the design - Explore catalog.

Reply:
Sure, your arguments sound very confident!